### PR TITLE
Development strategy -Home Page

### PR DESCRIPTION
**Pull request by minjupgeorge**
## General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work

### HTML

- [ ] pages besides `/index.html` are stored in `/pages`
- [ ] the code is well-formatted
- [ ] the HTML is valid
- [ ] there are no inline styles (example: `style='color: red;'`)
- [ ] there are no `<style>` tags with CSS, all styles are hrefs
- [ ] there is no inline JavaScript (example: `onclick='doSomething()'`)
- [ ] there are no `<script>` tags with JS, all JS is in an separate file
- [ ] `id`s are used for JavaScript only, not for CSS
- [ ] semantic tags are used
- [ ] spelling and grammar is correct in all site content

### CSS

- [ ] the code is well-formatted
- [ ] passes all the linting checks

### CODE REVIEW

File: planning/development-strategy.md [hunk 1]
```diff
@@ -1,39 +1,142 @@
-# Restaurants finder
+# Job - Advisor
+
+<!---minju edited--->
+
```
Comment (MMikhailova) [inline planning/development-strategy.md:4]: @minjupgeorge please change the description of the project so it's not only for IT sector.

File: planning/development-strategy.md [hunk 2]
```diff
@@ -1,39 +1,142 @@
...
+Job Advisor website is for motivated students and job seekers to help them to
+kickstart their career in IT and also search their dream job without
+registration.
 
 ---
 
-<!-- ## User Story Dependencies
+## Setup
```
Comment (MMikhailova) [inline planning/development-strategy.md:11]: @minjupgeorge It would be nice to add React, Mui, Strapi

File: planning/development-strategy.md [hunk 3]
```diff
@@ -1,39 +1,142 @@
...
+- styling to the top left of the main page.
 
 ---
 
-## 0.Setup
+### Navigation Bar
+
+> As a userI want to easily navigate the site
+
+- [ ] _There is a navigation bar with links to other pages of the site(About
```
Comment (MMikhailova) [inline planning/development-strategy.md:40]: Nav bar is should have. Please remove from here. Development strategy is only for must-haves

**Review by MMikhailova — CHANGES_REQUESTED** [review]

File: planning/development-strategy.md (hunk 1 above)
Comment (minjupgeorge) [reply planning/development-strategy.md:4]: done

File: planning/development-strategy.md (hunk 3 above)
Comment (minjupgeorge) [reply planning/development-strategy.md:40]: sure

File: planning/development-strategy.md (hunk 2 above)
Comment (minjupgeorge) [reply planning/development-strategy.md:11]: ok I will update it

**Comment by minjupgeorge** [comment]
@MMikhailova  I have made changes as you commented. am closing this Pullrequest. since I have cloned new repo and updated development strategy on the new repo

**Pull request closed without merging**